interfaces/IResourceType.ts

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Protocol from 'devtools-protocol';
+import Network = Protocol.Network;
+import DevtoolsResourceType = Network.ResourceType;
+
+export enum ResourceTypes {
```

Comment:
Is this supposed to be plural? Seems like most of our enums are singular.

Reply:
I'm not sure what our standard is here. KeyboardKeys is also exported from @hero/client and it's plural so I followed it.

I can go either way.

Reply:
Hmm. Yeah, that's the only one in the codebase if you search for `enum `

Reply:
Fixed to be singular.